backstop/tests/iModel-loader.html

```diff
@@ -0,0 +1,398 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>iModel loader | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .iui-demo-full-page {
+        height: 90vh;
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .iui-loading-imodel {
+        width: 250px;
+        margin-bottom: 20px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <section class="iui-demo-full-page">
+      <div
+        id="imodel-loader"
+        class="iui-progress-indicator-full-page"
+      >
+        <!-- Use iModel-loader.svg from @itwin/itwinui-illustrations package. -->
+        <svg
+          class="iui-loading-imodel"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 739.97 854.77"
+        >
+          <style>
+            .iui-loading-imodel-shape {
+              fill: rgb(83, 162, 26);
+              opacity: 0.5;
+            }
+
+            .iui-loading-imodel-i {
+              fill: #FFF;
+            }
+          </style>
+          <g>
+            <polygon
+              class="iui-loading-imodel-shape outer outer-12"
+              points="369.98 641.16 462.48 694.56 554.66 747.78 369.98 854.41 369.98 641.16"
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.67"
+              />
+            </polygon>
+            <polygon
+              class="iui-loading-imodel-shape outer outer-11"
```

Comment:
so wait.. all these classes needed??

Reply:
I can remove `outer` and `inner` classes, 
`iui-loading-imodel-shape` is used for styling.
`outer-11` is for locating polygons for timing calculations.

Reply:
or I can just rename classes if this is confusing.

Reply:
> `outer-11` is for locating polygons for timing calculations.

Are you sure? It was used when you had CSS animation but I can't find where it's used now

Reply:
It's not "used" used. It's just convenient to have these polygons numbered for manual calculations 😃

Reply:
so maybe remove those now as you have calculated everything you needed...?

Reply:
Does it really hurt having them? 😀 Because if some changes are needed, we will add them again just to know which polygon is which and then once again remove?